src/js/disability-benefits/containers/ClaimEstimationPage.jsx

```diff
@@ -16,7 +18,7 @@ export default class ClaimEstimationPage extends React.Component {
             <nav className="va-nav-breadcrumbs">
               <ul className="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
                 <li><Link to="your-claims">Your claims</Link></li>
-                <li><Link to={`your-claims/${this.props.params.id}/status`}>Your Disability Compensation Claim</Link></li>
+                <li><Link to={`your-claims/${this.props.params.id}/status`}>Your {getClaimType(this.props.claim)} Claim</Link></li>
```

Comment:
There's an issue here, similar to what you fixed on other pages: if you go to this page and refresh, you'll get a console error and the page won't load. You can add a check that `this.props.claim` exists before calling `getClaimType`, or add `loading` to `mapStateToProps` and check that.